src/components/Streak.tsx

```diff
@@ -0,0 +1,40 @@
+import { StreakDownArrowIcon, StreakUpArrowIcon } from './icons';
+
+const Streak = ({
+  streak,
+  fixedDigits = 0,
+  rightLabel = '',
+}: {
+  streak: number;
+  fixedDigits?: number;
+  rightLabel?: string;
+}) => {
+  if (streak === 0) {
+    return '';
+  }
+
+  if (streak === Number.NEGATIVE_INFINITY) {
```

Comment:
Negative Infinity used to represent N/A. It's good because it sorts well with the other values. Negative value though is valid, so using this as a symbolic value as well as one that sorts last worked out.